core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -1070,4 +1070,14 @@ public IntHolder(int value) {
             + "from \"s\".\"primesCustomBoxed\"")
         .returnsUnordered("EXPR$0=false\nEXPR$0=false\nEXPR$0=true");
   }
+
+  @Test public void testDecimalNegate() {
```

Comment:
Since CALCITE-3892, we made junit test classes and methods into non-public.

Reply:
Sorry, i miss this point